src/ui/compilationStatusView.ts

```diff
@@ -18,6 +18,8 @@ function toStatusMessage(status: CompilationStatus): string {
     return Messages.CompilationStatus.CompilationSucceeded;
   case CompilationStatus.VerificationStarted:
     return Messages.CompilationStatus.Verifying;
+  case CompilationStatus.VerificationInProgress:
+    return message ?? Messages.CompilationStatus.Verifying;
```

Comment:
The message will say something like this:

`Verifying CheckWellformed$$_module.__default.ToRelative ...`

which is arguably ugly.
Would you be able to remove the `CheckWellformed$$`, the `_module` if it is there (it's the default module) and  `__default` if it is there (it's the default class) ?

Reply:
Will do, this way we'll be in line with Emacs too.  In this PR or on the Dafny side?

Reply:
I think it's better to do it on the Dafny side, since else we make the IDE start to be aware of Boogie, which we would like to avoid.

Reply:
Perfect, will update the pull request right away